packages/cosmic-swingset/test/export-storage.test.ts

```diff
@@ -85,7 +100,10 @@ test('exportStorage example', t => {
 });
 
 test('exportStorage clears crufty ToyUSD PSM', t => {
-  const config = {
+  const config: {
+    clearStorageSubtrees: string[];
+    exportStorageSubtrees: string[];
+  } = {
```

Comment:
This just looks like noise; those match the inferred types.
```suggestion
  const config = {
```

Reply:
After fighting with Typescript for too long, I have a different suggestion: use a symbol for the TestDataKey, and all sorts of typing and namespace collision problems vanish.

[TypeScriptPlayground](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBA+jAvDAygTwLYCMQBsAUA5ACoCm0AIgIZRUDSpahAlANwBQUaADqTAAoBXLLgCWEABakAJgDkQ0vonYwYAbxWqYAbQDWALhjQATqLABzALqGhI8VLkLSHLTriWA-IcFhdYEADuYC4wAL7s7AD0kTAAkjDS4ISwfoEwEmlQIDAAZoK4uGgwXLwwAAb4slQYpAA0MACCxuYQzEgAfDAASqRQgsaQZQni3DTAEvVYgrBQEuIw5r0QMMC4IBCkAHScPHwAQmMSAMISVGYoWcZUiw388cgaqotQhvijs4YmZuZtiJ1fFhgAB8YGB8rgQqQwFBTORXu8JJ8Yd9fp0mlc0AAePSMJGmCz1ABuVFwglIXiMyIslnaIXGolw0mMUPhNERlPxPw6HO+2ksIQ2UAA6qJZiBpvIoKIcmhXpt5VDkXDGsYMdiEXjvkSSWSKQCrO1UTBCSBRNIOKEOCV9ocTmcwBcQFdFpjNABZGCkAAeUCh0mWukYIByMAOUHGdvOl2upFu9xggbQwdDttOUadMbjtXYnWQ+E0NTF0kMbuzqiozQgNgr1V6pGMEExYYjaYd0Zud20bpp2aNPT6A2Iuybqftjudsc73dpEVAkFgGCogebx1b45jSBgb2EYkkMhsO-sMnkiiNjxW4GgCVIuDrxZ5Fj5m75dMvsDWIF0gm4m7ebM1FjMAedh7o4ijAqC4Lcueqi3rA-jgcg3CHqBISqNKW4IkgiDIIQyEgQ4LAwMyA5gKCThobkTpbnOV6JjAyYIpszLcLgVDAKQ+CRAAevhu4OAAOpskT1IQLCbBArGikQmwsG0MFaBh+DWsmCF8AAhDhMCECAWAAFakMAUCEMCIJqdhyBggUbQkf0ZFWRCmiuOZllODiaD8k5ikhvg6lqTZvR2ZBBSUeEri2QM5GKCEloRKotGwIqsLLHmGoPj8hjuQB5jaqS5LZZYT5-OoXkJVFSgwB+X7cH+sxsF5Sl+U4AWkTonmuGV4ovDo+r1PqHiFZYz7tVoZX0oy1C0JuADy+mGVAmxJaI5D4P5MBUMs6JUFi2i9QIKEOCepA0pROTUfgZXufU43SENyY3ZNVDyV56E+TdmwIJpyA+IoORmDIz2uK4XWbMhkj4Fd5QACRqDd2juKEZSWPVQNhJ6uAbCVqOqCDYMSBDibI5RqhhVopOqBFZFdTFHCaGVWCjumE6GCukZthmiy-oWGTSPUFYtGeXkQAEorjFu3MKIDQPABtfCEM8hCGApHVvjoCJDcg-PLBtPVUlYxOjarLmVSAn7frVEgo9jSkuVpDktUFDkG69W5qR9FkwD9pB-WAAPEYFkVOy9WiU9DahqfDliIwb5OjbL2lLeQitY9jl3q5uWvrcsu16yNqOh4nEAW1brix-F8eEDdzJgMnyuG-Oatshr62VlnuucnnQMF9CyXF5si41QTjDI9yiYl2Twcy5jhCCiKYoSiAUoyrXweqGdxg0ar2jp8mmc6znHd8lL2PXnBMig4I4MIuPper-7pEx8Hv1UPkLx37MxhpAAoqqTr4GU-hYCiAwKxQwMMJbSERuPUmsUKYBzImoGADNwyrjHO2Oop87xhAtLTMq3UqBgCKA8WKURIi+mgEQb03AnRQHXJzb01RWKkEIPUWAxUNDnjKlQmhdDSAoGEDCUgyp9RFR0HhA659gCbBAIQEaXCvS8CMvuduWp0oDVEdoLy2hxEEUkdIlh2ksjcFkWWLQ2i+JHmkJsKRIBNiiBMVonR-E9G2NEJsGuokAAsAAmBxrhzESKsTYzYAQDGEAwKbPxMBPJ01Vog5BLY0Ecz4KETci5lyM3ZhOfAClgBK2DiAfJJ9I6GGSCAYxpjUaiCKSfUESs3DWBgD4sIlSgahFaa4AI9SSnaQiTIlpwd2leSGSTK2ZV2J9BJJubhxhaHoPwAk1BTMYz1BmXM5J-CsCCPIFbBaihSDcC-gAR0ECSfAEzTm4FWQo+aAMLQwGiPVUh5CoCUIUTw9BKxbwVmWMAYwggchcBgMQEAaAACqKAKACBQG6AxbDOgcNiQ3Ocf1zA1Pit84wvDNnbKrOlF8Xk1nYoEcyYResCUk03LkzFxKtmkrxQE3R0gomqCJegnF9LMrBycZY0GEAMCbFiCgYgLDuUWNAiEkkcFRWox5RK7gn9CTiFEOAfgptcD916KYYAEAZVaEsKYkh8VVbiocG6e0wFnFgQqgpa4TpRDACqDUPFdcsBXDAPeRBPTtDaKFSK+o8SQAVmkLECgpScDBoAAwAEZvEAGZPEmVCJYIaIyyatO4Py9FWg-XZtcCC8FkK81A0LNql1DT6mSBAHw0QAAvUghg41ozTajFt6bhmtICFK3oxa7XatjXG7p7hSlQ1HUm1pbaFUgCVRAFVYA1V4GLaWh15aemIKyLQXAbozC+mkEccAhJ6y7sMN4yNzaJ2Gtpsahu8TMm8PqIoM+0g0bIHSaQVma55mmpkOaswYzVYXKmcgNlyT8yuEWWze9pVwCosWu82ZtLcWtJRaIcw1iaXspJUIiApjdmbH2Yck5ZzANXJ0I479VjM0Cr9SysRFG+XUeFZsAtEKKC0cZVahjgqmMschZq5EOqwnaF0gZIyMBZqiagJYdjcqHCSoKL0GT9Gu0KYWn2h1A6wmjqhkpwJoNFXKtVeq-jZahMifmuJuaRlpMGs0ETOKxRNgwjJPgR9d47FgFWIIRQRdZPnyo9x4gzHQWsbkvcx5tM8iealOAT08H1nZPlJsLW3hfD+CCEfQwBCihqHCEAA)